css-align-3/resources/alignment-parsing-utils.js

```diff
@@ -0,0 +1,19 @@
+var position = [ "start", "end", "left", "right", "center", "flex-start", "flex-end"];
+var distribution = [ "stretch", "space-around", "space-between", "space-evenly"];
+var baseline = [ "baseline", "last-baseline"];
+
+function checkPlaceContent(alignVal, justifyVal)
```

Comment:
That's not correct for the function I'm implementing. 
I need the precise values passed as argument to construct the style rule:

   div.setAttribute("style", "place-content: " + alignVal + " " + justifyVal)

Then, the computed value is the one that have the modified justifyVal variable.